sources/@roots/bud/src/seed.ts

```diff
@@ -93,8 +93,8 @@ export const seed: Partial<Store.Repository> = {
    * @public
    */
   patterns: {
-    js: /\.(js|jsx)$/,
-    ts: /\.(ts|tsx)$/,
+    js: /\.([c|m]?jsx?)$/,
```

Comment:
Why not just enumerate them more explicitly? `\.(mjs|cjs|js|jsx)$` Then it gets rid of false positives (easier to read too). But either way, looks good 👍